File: Baekjoon_Javascript/5355.js
let input = require('fs').readFileSync('/dev/stdin').toString();

let [T, ...cases] = input.split('\r\n');
const solution = (cmd) => {
    let result = +cmd[0];
    for(let i = 1; i < cmd.length; i++) {
        switch (cmd[i]) {
            case '@': result *= 3; break;
            case '%': result += 5; break;
            case '#': result -= 7; break;
        }
    }
    return result;
}

let results = cases.map(v => solution(v.split(' ')).toFixed(2));

console.log(results.join('\n'));

File: Baekjoon_Python/Baekjoon_Python/5355.py
import sys

cmd = {'@':lambda x:x*3, '%':lambda x:x+5, '#':lambda x:x-7}
answers = []
input()

for x in sys.stdin.readlines():
    x = x.split()
    result = float(x[0])
    for i in x[1:]:
        result = cmd[i](result)
    answers.append(f'{result:.2f}')

print('\n'.join(answers))